libs/langchain-community/src/vectorstores/upstash.ts

```diff
@@ -78,10 +96,14 @@ export class UpstashVectorStore extends VectorStore {
    */
   async addDocuments(
     documents: DocumentInterface[],
-    options?: { ids?: string[] }
+    options?: { ids?: string[]; UpstashEmbeddings?: boolean }
```

Comment:
Small style nit - we generally use camel case for properties. I can push a rename